src/common/points.h

```diff
@@ -584,9 +584,9 @@ inline static void gen_rand_array(struct sfmt_state_t *s, w128_t *array, int siz
 #if defined(__BIG_ENDIAN__) && !defined(__amd64) && !defined(BIG_ENDIAN64)
 #define BIG_ENDIAN64 1
 #endif
-#if defined(HAVE_ALTIVEC) && !defined(BIG_ENDIAN64)
```

Comment:
Similarly, this is appears to be a copy-paste from some upstream header.
The change probably needs to be done upstream first.